status-im/status-react
# Enable pairing & contact recovery

This PR enables pairing outside of dev-mode and contact-recovery, which
is useful in the case a new device is added or re-installed.

To test pairing, the same applies to previous builds, but it everything should just be enabled by default.
To test contact recovery:

1) Create account on A1
2) Create account on B1
3) Create a group chat between A & B, exchange a few messages
4) Recover account on B2
5) Once the account is recovered, send a group chat message from A to B
6) You should see the message on B2

On nightly the last message should not be received by B2, you would have to create a group chat on B2 and include A, at that point the devices are able to communicate. 

It should help with https://github.com/status-im/status-react/issues/7362 although we still need logs to understand exactly the issue.


status: ready
